docs/index.rst

```diff
@@ -1,89 +1,56 @@
-That said, these functions will also work with any interactive backend (e.g. `%matplotlib qt5`).
+* Better understand a function's change with respect to a parameter.
+* Visualize your data interactively.
 
-Generic
-^^^^^^^
-Provides a way to interact with Matplotlib that will function outside of a Jupyter notebook. 
-Three new features are created:
+To achieve this, mpl_interactions provides:
 
-* A very niche way to compare 2D heatmaps 
-* Scroll to zoom
-* Middle click to pan
+* A Matplotlib/ipympl-aware interact function for Jupyter notebooks.
+* A function to compare horizontal and vertical slices of heatmaps.
+* A function allowing the user to zoom using the scroll wheel.
 
-Again, this module is also compatible with any backend. 
+Installation
+^^^^^^^^^^^^^
 
-Utils
-^^^^^
-This module includes utility functions to make user experience just that little bit easier. 
+To install, simply run:
 
-* `ioff` as a context manager
+``pip install mpl_interactions``
 
-.. code-block:: python
-
-   from mpl_interactions.utils import ioff
-   with ioff:
-      # interactive mode will be off
-      fig = plt.figure()
-      # other stuff
-   # interactive mode will be on
-
-* `figure` that accepts a scalar for `figsize` (this will scale the default dimensions)
-
-.. code-block:: python
+Further instructions for installation from JupyterLab can be found on the :ref:`Installation` page. 
 
-   from mpl_interactions.utils import figure
-   fig = figure(3)
-   # the default figsize is [6.4, 4.8], this figure will have figsize = [6.4 *3, 4.8 *3]
+Basic example
+^^^^^^^^^^^^^^
 
-* `nearest_idx` allows you to avoid ever having to write `np.argmin(np.abs(arr - value))` ever again
-
-Jupyter
-^^^^^^^
-Provides a different approach than `ipywidgets.interact` for making sliders that affect a Matplotlib plot. 
-When using `ipywidgets.interact` you are responsible for:
-
-1. Defining the function to plot `f(x, ...) => y`
-2. Handling the plotting logic (`plt.plot`, `fig.cla`, `ax.set_ylim`, etc.)
-
-In contrast, with mpl-interactions, you only need to provide `f(x, ...) => y` to have the plotting and 
-updating boilerplate handled for you. 
+To control a plot with a slider:
 
 .. code-block:: python
 
-   x = np.linspace(0,6,100)
-   beta = np.linspace(0,5*np.pi)
-   def f(x, beta):
-      return np.sin(x*4+beta)
-   interactive_plot(f, x=x, beta=beta)
-
-Follow the links below for further information on installation, functions, and plot examples.
-
-.. toctree::
-   :maxdepth: 3
+   %matplotlib ipympl
+   from mpl_interactions import interactive_plot
+   x = np.linspace(0,np.pi,100)
+   τ = np.linspace(.5, 10, 100)
+   β = np.linspace(1, 10, 100)
+   def f1(x, τ, β):
+       return np.sin(x*τ)*x*β
+   def f2(x, τ, β):
+       return np.sin(x*β)*x*τ
+   fig, ax, sliders = interactive_plot([f1, f2], x=x, τ = τ, β = β)
+   _ = plt.legend()
 
-   Installation
-   Backends
-   compare-to-ipywidgets
-   API
-   Examples
-   Contributing
-   
```

Comment:
We need to keep this toctree otherwise we lose the contents of the navigation sidebar

Reply:
you can just add it back in below the example